### ISSUE TITLE
Hide currently open menu without knowing trigger

### ISSUE THREAD

**Issue by dtex** [swisnl/jQuery-contextMenu#248]
I'd like a command I can call to hide whatever menu might already be open.

Triggering a new menu calls this:

````
$currentTrigger.data('contextMenu').$menu.trigger('contextmenu:hide');
````

I'd like to be able to call that from my own code (when another non-context menu UI element is triggered)

I could keep track of the current trigger in my own code but it be nice of the feature was here. 

If this seems like a good addition I'll submit a PR, otherwise close it and I'll handle it on my end.

**Comment by bbrala** [comment]
I think this might be fixed should PR #243 is merged which would add:

$(document).on('mousedown.contextMenu.global', function(){});

Thank pull request is a bit of a mess thought, it needs some cleaning.

**Comment by bbrala** [comment]
You could use ```$.contextMenu.menus``` which contains all the menu, is not really documented though, so cannot give guarantees :)
